CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@
 - Check the [open issues](https://github.com/okTurtles/group-income/issues).
 - View the [project board](https://github.com/okTurtles/group-income/projects).
 - Check especially what's [`up for grabs`](../../labels/Note%3AUp-for-grabs), [`high priority`](../../labels/Priority%3AHigh), [`level: starter`](../../labels/Level%3AStarter), or has a **[`Bounty`](https://github.com/okTurtles/group-income/issues?q=is%3Aopen+is%3Aissue+label%3ANote%3ABounty)**
```

Comment:
Is removal of these intentional. If not, I suggest hardcoding the absolute URL.

Reply:
Removal of what...? These lines are unmodified...

Reply:
Sorry, I saw the `-` and wrongly believed they were deleted (my setup where I say this doesn't use colours). However, are you sure that the links will work on the GH pages site? It might still be worth it to set them to absolute URLs.

Reply:
OK, sure that's a good suggestion, just pushed a fix